content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -57,16 +10,22 @@ Register your Amazon account in Odoo
 To register your seller account in Odoo, navigate to :menuselection:`Sales --> Configuration
 --> Settings --> Connectors --> Amazon Sync --> Amazon Accounts` and click on **CREATE**.
 
-The **Seller ID** can be found in Seller Central under the link **Your Merchant Token** on the
-**Seller Account Information** page. The **Authorization Token** is the one you generated in the
-:ref:`previous step <amazon/generate_auth_token>`.
+#. Choose a meaningful name for your account (e.g. `Europe`), select your :guilabel:`Base
+   Marketplace`. This is the original sign-up marketplace of your seller account (e.g. if you
+   created your account on Amazon Germany, your base marketplace will be `amazon.de`)
 
-Upon saving, your credentials are checked. In case of issues, an error will be displayed - the
-information cannot be saved until your credentials are recognized by Amazon.
+#. Upon saving, a button to :guilabel:`link with Amazon` appears. Click on it to be redirected to
+   the Amazon login page, or directly to the consent page if you are already logged. There, you
+   confirm that you direct Amazon to provide Odoo the access to your account and related data.
```

Comment:
Changed wording to make it flow better.

`confirm that you want Amazon to give Odoo access to your account and related data.`